packages/react/src/components/SkipLink/SkipLink.tsx

```diff
@@ -16,11 +16,7 @@ export const SkipLink = ({
   ...rest
 }: SkipLinkProps): JSX.Element => {
   return (
-    <a
-      href={href}
-      className={cl('ds-skiplink', className)}
-      {...rest}
-    >
+    <a href={href} className={cl('ds-skiplink', className)} {...rest}>
```

Comment:
_Bare_ prik 😇 
```suggestion
    <a className={cl('ds-skiplink', className)} {...rest}>
```
...også ikke plukke den ut av args? ☺️

Reply:
Trur det var ein lint regel me hadde tidligare som ikkje fekk med seg href i `...rest`, ser ut til å funke no - så tar bort!